modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/PageSupport.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.pagememory;
+
+import org.apache.ignite.internal.pagememory.metric.IoStatisticsHolder;
+import org.apache.ignite.lang.IgniteInternalCheckedException;
+
+/**
+ * Supports operations on pages.
+ */
+public interface PageSupport {
+    /**
+     * Returns an absolute pointer to a page, associated with the given page ID. Each pointer obtained with this method must be released by calling
+     * {@link #releasePage(int, long, long)}. This method will allocate a page with the given ID if it doesn't exist.
+     *
+     * @param groupId Group ID.
+     * @param pageId  Page ID.
+     * @return Page pointer.
+     * @throws IgniteInternalCheckedException If failed.
+     */
+    public long acquirePage(int groupId, long pageId) throws IgniteInternalCheckedException;
```

Comment:
A nitpick: `public` is redundant in interfaces. Is it retained on purpose?

Reply:
I'll remove it, forgot to do it. "public" was mandatory according to old code convention in Ignite 2.x